webpack.config.js

```diff
@@ -0,0 +1,71 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+const path = require('path');
+const HtmlWebpackPlugin = require('html-webpack-plugin');
```

Comment:
When I run `webpack serve` command I got this error:
```js
URIError: Failed to decode param '/%PUBLIC_URL%/manifest.json'
```

It will be good to fix it.